geary-papermc-features/src/main/kotlin/com/mineinabyss/geary/papermc/features/items/resourcepacks/ResourcePackGenerator.kt

```diff
@@ -57,7 +62,15 @@ class ResourcePackGenerator(world: Geary) : Geary by world {
                 .addTo(resourcePack)
         }
 
+        config.jarResources
+            .forEach {
+                resourcePack.unknownFile(it.path, Writable.resource(it.classLoader, it.resource))
```

Comment:
should use a when statement, check filepath and accordingly use the correct method, and unknownFile in else